DigitalLearningSolutions.Web/Models/Enums/DelegateAccessRoute.cs

```diff
@@ -10,6 +10,12 @@ public class DelegateAccessRoute : Enumeration
         public static readonly DelegateAccessRoute ViewDelegate =
             new DelegateAccessRoute(1, "ViewDelegate");
 
+        public static readonly DelegateAccessRoute TopCourses =
+            new DelegateAccessRoute(2, "TopCourses");
+
+        public static readonly DelegateAccessRoute DelegateCourses =
+            new DelegateAccessRoute(3, "DelegateCourses");
+

```

Comment:
The idea of the `DelegateAccessRoute` was to help manage pages that could be linked to from multiple places, and needed to navigate back to those pages. E.g. the Delegate Progress page and its child pages could be accessed from the CourseDelegates page or the ViewDelegate page.

This was then used as an easy sign of which back links/breadcrumbs we should use on a page. I think I can see why this has been implemented, but I reckon it is unnecessary. Don't know if this is something Kevin has asked for though.

Also doesn't look like this implementation has much validation on it. So we it might be possible to reach the the CourseDelegates page with a CourseDelegates access route, which is pretty nonsensical to me.